doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -516,3 +516,123 @@ arguments used for configuration.  Here is a table describing these emulated dev
    * - ``wdt-i6300esb``
      - Emulated i6300ESB PCI Watch Dog Timer (WDT) Intel processors use to
        monitor User VMs.
+
+Launch Script
+*************
+
+Launch script is used to launch a User VM from the ServiceVM command line. It
+is generated by configurator according to several settings for a User VM. With
+launch script, user does not bother tweaking acrn-dm parameters.
+
+Most settings for User VMs are used at launch time. They are first stored into
+``scenario.xml`` and then parsed by
+``misc/config_tools/launch_config/launch_cfg_gen.py``
+to add shell commands into launch script according to the template
+``misc/config_tools/launch_config/launch_script_template.sh``.
+The template defines following helper functions to do system settings or to
+generate an acrn-dm command option. For details about all acrn-dm parameters,
+please refer to "Device Model Parameters".
+
+``probe_modules``
+    Install necessary modules before launching a Post-launched VM. For
+    example, pci_stub is used to provide a stub pci driver which does nothing
+    on attached pci devices. and passthrough
+    PCIe device will be unbind from original driver and bind to the stub, so
+    that it can be safely controlled by Post-launched VM. TODO
+
+``offline_cpus <cpu_apicid>...``
+    This is called if we are launching an RTVM or VM whose scheduler is
+    "SCHED_NOOP". In both situations, CPU sharing between multiple VMs are
+    prevented.
+    This function will trigger linux cpu offline first, and then inform
+    hypervisor through HSM. Hypervisor will offline VCPU and freeze VCPU
+    thread.
+
+``unbind_device <bdf>``
+    Unbind a PCIe device with specified BDF (bus, device and function) number
+    from its original driver and re-bind it to the pci-stub driver. After that
+    linux will not operate on that device any more and it can be passthrough
+    to User VM safely.
+
+``create_tap <tap>``
+    Create or reuse the tap interface which is attached to acrn-br0 bridge.
+    acrn-br0 is registered to systemd-networkd.service after installing acrn
+    deb package. This is used to add ``virtio-net`` interface to User VM.
+    ``virtio-net`` interfaces for all User VMs are virtually connected to a
+    subnet behind the acrn bridge.
+
+``mount_partition <partition>``
+    Mount specified partition to a temporary directory created by ``mktemp -d``,
```

Comment:
Configurator allows user to give a rootfs file inside an image file as virtio-blk backend. The typical usage is assigning the "/boot/initrd.img*" inside a raw image file to User VM. The format of the string in configurator virtio-blk input box is "<imgfile>:<path/to/rootfs>". The generated launch script will mount the <imgfile> to a temporary directory first so that acrn-dm could access the inner rootfs. After the acrn-dm instance exits, unmount_partition will be called.

May I add this explanation ?